machines/bleShare/request/requestMachine.ts

```diff
@@ -803,6 +811,22 @@ export const requestMachine =
             ),
           );
         },
+
+        sendVcDataIntent: context => {
+          const {verifiableCredential} = context.incomingVc;
+          const {credentialSubject: subject} = verifiableCredential;
+
+          ODKIntentModule.sendBundleResult({
+            // biometrics: subject.biometrics,
```

Comment:
Biometrics string data was sent in the original implementation but I noticed there's a performance impact due to the size of data saved on the ODK Collect app so I commented it out for now and is subject for review.